project/make/.bundle

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# helper scripts for creating package bundles
```

Comment:
Could we call this file `.packages` or something like that instead?  Each script in `project/make/*` is referred to as a "bundle script", and I don't want to overload that term any more than we already have. :+1: